From c27eef89a1ce1a298e43a02b12e92e7392ba7925 Mon Sep 17 00:00:00 2001
From: Wei-Chiu Chuang <weichiu@cloudera.com>
Date: Mon, 2 May 2016 06:42:16 -0700
Subject: [PATCH 1486/2748] HADOOP-12805. Annotate CanUnbuffer with
 @InterfaceAudience.Public (Contributed by Ted Yu)

(cherry picked from commit 4b0e59fdc010be517dd67ddf35c962da204df547)
(cherry picked from commit e79a5e4460b5199eebb0d788fdc1eaa36d6b0ac4)
(cherry picked from commit 89e90becfc667c330e7460aa8dddb02390770681)

Change-Id: I810c92d8c3f6f8f4c0e9ac329cadd0c0a19985fa
---
 .../java/org/apache/hadoop/fs/CanUnbuffer.java     |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/CanUnbuffer.java b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/CanUnbuffer.java
index 07e65f5..e322798 100644
--- a/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/CanUnbuffer.java
+++ b/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/CanUnbuffer.java
@@ -25,7 +25,7 @@
  * FSDataInputStreams implement this interface to indicate that they can clear
  * their buffers on request.
  */
-@InterfaceAudience.Private
+@InterfaceAudience.Public
 @InterfaceStability.Evolving
 public interface CanUnbuffer {
   /**
-- 
1.7.9.5

